icons/hardhat/hardhat-original.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="256px" height="177px" viewBox="0 0 256 177" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="xMidYMid">
```

Comment:
Viewbox needs to be 0 0 128 128.
See [SVG STANDARDS](https://github.com/devicons/devicon/wiki/SVG-Standards) and [Recommended resources and tools](https://github.com/devicons/devicon/wiki/Recommended-resources-and-tools) for more info

Reply:
Sorry my fault, now it's centered..